src/mrpro/data/KData.py

```diff
@@ -162,6 +162,9 @@ def from_file(
             additional_fields=('center_sample', 'number_of_samples', 'discard_pre', 'discard_post'),
             convert_time_stamp=convert_time_stamp,
         )
+        # discard_pre and discard_post is of shape (N_readouts,1,1,1,1) and we only need it as (N_readouts,)
+        discard_pre = discard_pre[:, 0, 0, 0, 0]
```

Comment:
instead of the comment, maybe 
`discard_pre.squeeze_()`
or
discard_pre = rearrange("discard_pre, "readout 1 1 1 1 -> readout")
discard_post = rearrange("discard_pre, "readout 1 1 1 1 -> readout")

Reply:
if it is easy to add chaging discards to the test data, could you do it? 
if it would take more than 10min / 20 lines, it's ok to leave it